charmcraft/main.py

```diff
@@ -46,6 +46,15 @@
 ]
 
 
+def _get_option(args, name):
+    """Get a general option, merging with OR both global and command versions.
```

Comment:
"merging with OR" seems to be redundant. I suppose it meant "Get the global or command option, including both if present'.

Reply:
Improved the docstring, thanks!